### PR TITLE
fix(doc): update server and port when serving TGI

### PR DESCRIPTION
**Pull request by tengomucho**
# What does this PR do?

Correct server and port in docs for serving TGI.
Note that port redirection is removed because we use `--net=host`.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you make sure to update the documentation with your changes?

### CODE REVIEW

**Comment by tengomucho** [comment]
@ManfeiBai I know you opened https://github.com/huggingface/optimum-tpu/pull/38 before, but I think this might be a cleaner solution, please let me know if it is fine for you.

**Review by mfuntowicz — APPROVED** [review]

**Review by ManfeiBai — APPROVED** [review]
Thanks, LGTM

**Comment by ManfeiBai** [comment]
> @ManfeiBai I know you opened #38 before, but I think this might be a cleaner solution, please let me know if it is fine for you.

thanks, this looks more clean, and let's merge this PR, would close #38 after this PR merge

**Pull request merged**
